include/rabbit.hrl

```diff
@@ -120,12 +120,15 @@
 
 -record(message_properties, {expiry, needs_confirming = false, size}).
 
--record(plugin, {name,          %% atom()
-                 version,       %% string()
-                 description,   %% string()
-                 type,          %% 'ez' or 'dir'
-                 dependencies,  %% [{atom(), string()}]
-                 location}).    %% string()
+-record(plugin, {name,             %% atom()
+                 version,          %% string()
+                 description,      %% string()
+                 type,             %% 'ez' or 'dir'
+                 dependencies,     %% [{atom(), string()}]
+                 location,         %% string()
+                 plugins_versions, %% [{atom(), string()}]
```

Comment:
I'd also use an example in the comments. Type spec-ish comments are useful but so are specific examples.